gammapy/time/lightcurve.py

```diff
@@ -320,6 +321,124 @@ def create_fixed_time_bin(time_step, spectrum_extraction):
             ])
         return intervals
 
+
+    def create_fixed_significance_bin_lc(self, significance, significance_method, energy_range,
+                                            spectrum_extraction, separators=[]):
+        """
+        Create time intervals such that each bin of a light curve reach a given significance
+        Parameters
+        ----------
+        significance: float
```

Comment:
Add space before the `:` or Sphinx won't format the docstrig correctly. Here and for all other parameters.